java/samples/src/main/java/RocksDBSample.java

```diff
@@ -31,6 +31,7 @@ public static void main(final String[] args) {
          final Filter bloomFilter = new BloomFilter(10);
          final ReadOptions readOptions = new ReadOptions()
              .setFillCache(false);
+         final Statistics stats = new Statistics();
```

Comment:
Looks good. However `Statistics` is a `RocksObject` and so should ideally be manually closed, along with `ReadOptions`. Probably best to put them in a `try-with-resources`